src/Owin.Security.Providers.SlackProvider/SlackAuthenticationHandler.cs

```diff
@@ -95,6 +95,20 @@ public SlackAuthenticationHandler(HttpClient httpClient, ILogger logger)
                 dynamic response = JsonConvert.DeserializeObject<dynamic>(text);
                 var accessToken = (string)response.access_token;
                 var scope = (string)response.scope;
+                dynamic bot = null;
+                dynamic webhook = null;
+                //According to https://api.slack.com/docs/oauth  if you specified bot in the scope request the bot creds will be passed back here
+                //we'll store them here and add these values to the claims below.
+                if (!string.IsNullOrWhiteSpace(scope) && scope.IndexOf("bot", StringComparison.InvariantCultureIgnoreCase) > 0)
```

Comment:
instead of setting the bot, and webhook properties here then checking them further below, can we just move these if statements to lines 128 and 133, and get rid of the two dynamic properties.